templates/next/utils/mercure.ts

```diff
@@ -0,0 +1,58 @@
+import has from "lodash/has";
+import { useEffect, useState } from "react";
+import { PagedCollection } from "../types/Collection";
+import { normalize } from "./dataAccess";
+
+const mercureSubscribe = (hubURL: string, data: unknown | PagedCollection<unknown>, setData: (data: unknown) => void) => {
```

Comment:
`data: unknown | PagedCollection<unknown>` is equivalent to `data: unknown`

I guess it should be `data: PagedCollection<unknown>`

Reply:
`data: PagedCollection<unknown>` would be invalid cause the data could be a PagedCollection or any other type

Reply:
We must also support the case when Mercure isn't enabled and so there is no Hub URL.